src/state/util.ts

```diff
@@ -40,7 +40,11 @@ export function Create(): AtemState {
 		},
 		settings: {
 			multiViewers: [],
-			videoMode: 0
+			videoMode: 0,
+			mediaPool: {
```

Comment:
I would prefer the mediaPool object to be optional, so that it is undefined rather than filled with zeros when we havent received the command

Reply:
Done!